docs/writing-tests/addon-vitest.mdx

```diff
@@ -0,0 +1,346 @@
+
+TK - Screenshot of test output with links to SB
+
+## Usage
+
+There are three primary ways to run tests using the Vitest plugin:
+
+### CLI
+
+The plugin transforms your stories into real Vitest tests, so you run those tests just like you run any other Vitest tests in your project. Typically, you will have a `test` script in your `package.json` that runs Vitest. When you run that script, the addon will find and run your story-based tests. Here's an example of running your tests (in [watch mode](https://vitest.dev/guide/cli.html#vitest-watch), by default) using the Vitest CLI:
+
+{/* TODO: Snippetize */}
+```sh
+npm run test
+```
+
+### Editor extension
+
+Transforming your stories into Vitest tests with the plugin also enables you to run and debug tests using Vitest [IDE integrations](https://vitest.dev/guide/ide.html). This allows you to run tests directly from your editor, such as VSCode and JetBrains IDE.
+
+TK - Screenshot of VS Code
+
+### In CI
+
+For the most part, running your Storybook tests in CI is done [via the CLI](#cli). However, to have the test output link to your published Storybook on test failures, you need to provide the [`storybookUrl` option](#storybookurl) in the plugin configuration.
+
+Here's an example using GitHub Actions. The steps are similar for other CI providers, though details in the syntax or configuration may vary.
+
+First, we run a command to build and publish Storybook. In this case, we'll use Chromatic. This gives us a URL to the published Storybook instance. We then pass that URL to the plugin configuration using an environment variable. Finally, we update the plugin configuration to use that environment variable in the `storybookUrl` option.
+
+```yaml
+TK
+```
+
+```js title="vitest.workspace.ts"
+process.env.SB_URL
+```
+
+## Configuration
+
+Most of the configuration for the Vitest plugin's behavior is done in the Vitest configuration files. However, you can also define configuration in your stories themselves, using [tags](../writing-stories/tags.mdx), to control how they are tested.
+
+In this example, the Default story will not be tested, and the Primary story will.
+
+{/* TODO: Snippetize */}
+```js title="Button.stories.tsx"
+import { Button } from './Button'
+
+export default {
+  component: Button,
+  // 👇 Apply `test` tag to all stories in this file
+  tags: ['test'],
+}
+
+export const Default = {
+  // 👇 Remove `test` tag from this story
+  tags: ['!test'],
+}
+
+export const Primary = {
+  args: { primary: true }
+}
+```
+
+By default, the plugin will run all stories with the `test` tag. You can adjust this behavior by providing the [`tags` option](#tags) in the plugin configuration. This allows you to include, exclude, or skip stories based on their tags.
```

Comment:
this feels a bit redundant, maybe we merge this on top with the text before the previous snippet? on line 177